.github/workflows/windows.yml

```diff
@@ -480,12 +480,8 @@ jobs:
         timeout-minutes: 12
         run: |
           export TFLAGS='-j14 !TFTP !MQTT !WebSockets !SMTP !FTP ${{ matrix.tflags }}'
-          if [[ '${{ matrix.config }}' = *'-DUSE_WIN32_IDN=ON'* ]]; then
-            if [[ '${{ matrix.config }}' != *'-DENABLE_UNICODE=ON'* ]]; then
-              TFLAGS+=' ~1448 ~2046 ~2047'
-            fi
-          elif [[ '${{ matrix.config }}' = *'-DUSE_LIBIDN2=ON'* ]]; then
-            TFLAGS+=' ~165 ~1448 ~2046 ~2047'
+          if [[ '${{ matrix.config }}' != *'-DENABLE_UNICODE=ON'* ]]; then
+            TFLAGS+=' ~165 ~1448 ~1560 ~2046 ~2047'  # IDN tests failing without Unicode. 165 is for libidn2, for WinIDN it is disabled globally
```

Comment:
Is it not better to maintain these ignores more globally in DISABLED?

I think it makes it easier to get an overview of what is ignored, plus if these are true problems with combinations then surely others who run the tests also want the same ignores?

Reply:
Agreed, it'd be better. I did not know that the global disables till just recently.

I'll give it a try.

Reply:
After testing: It's OK, but different than putting tests on ignore. With `DISABLED` the test disappears from the log without a trace. I'd be useful to see it there to verify that it's actually disabled. (In this case it wasn't, until adding `libidn2` to the `$feature` map in `runtests.pl`.)

Also tflags-ignore _runs_ the tests, making it easy to inspect what's wrong with it.

`DISABLED` seems suited for test with no intent to fix. I think these aren't like that, though fixing some of them is non-trivial (Unicode.)